docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
+      "text/plain": [
+       "<Figure size 1500x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# NOTE: this will consume an API request!\n",
+    "res = live.advanced_pv_power(resource_id)\n",
+    "timeseries[\"updated\"] = res.to_pandas()\n",
+    "pd.concat(\n",
+    "    timeseries.values(),\n",
+    "    keys=timeseries.keys(),\n",
+    "    axis=\"columns\",\n",
+    "    names=[\"Version\", \"Parameter\"]\n",
+    ").plot(figsize=[15, 4])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5fd7140-0cd0-41e4-986a-8c16ffd3ff0c",
+   "metadata": {},
+   "source": [
+    "## Adjusting Advanced PV Power Settings\n",
+    "\n",
+    "When comparing observed production to Solcast live PV estimates, if you find discrepancies you may be able to choose more appropriate [model settings](https://docs.solcast.com.au/?#0c9d3ccf-e2a4-4583-86a3-f89c8d658fde) to get estimates which better fit the observations.  We will give a couple of examples.\n",
+    "\n",
+    "### Snow Soiling\n",
+    "\n",
+    "In the example shown here, say our observed production reached 640 kW on the last day.  It turns out that there was snow at this location in the last few days, and the PV model by default applies a snow soiling adjustment in these conditions.  However, we know that we have kept the snow off our panels.  By turning off the snow soiling in the PV model as follows, we get estimates which now match our observed peak production."
```

Comment:
`However, we know that we have kept the snow off our panels.` suggest changing this to `we know that we just cleared the snow from our panels` as it is more the common expression used to describe why this feature exists